lib/librte_hash/rte_cuckoo_hash.c

```diff
@@ -1342,3 +1345,127 @@ rte_hash_iterate(const struct rte_hash *h, const void **key, void **data, uint32
 
 	return position - 1;
 }
+VERSION_SYMBOL(rte_hash_iterate, _v21, 2.1);
+
+int32_t __rte_experimental
+rte_hash_iterate_v1811(const struct rte_hash *h,
```

Comment:
Drop `__rte_experimental`, `rte_hash_iterate()` is not experimental. Review section "3.8.1. Updating a public API" of https://doc.dpdk.org/guides/contributing/versioning.html for what we need to do to version `rte_hash_iterate()`.